### PR TITLE
docs: main page updates around support

### PR DESCRIPTION
**Pull request by dreamorosi**
## Description of your changes

As described in #1132 there are some changes related to how to support the project that were made in the Python version of Powertools and that needed to be replicated here. This PR addresses that action item.

This PR also cover some minor housekeeping in 3 more sections of the docs.

### How to verify this change

You can verify this change by running `docs-buildDockerImage` & `docs-runLocalDocker` while in the root folder (requires Docker).

Or check the screenshots provided below (only for main page):
<details>

Screenshot 2022-10-25 at 07 40 37

Screenshot 2022-10-25 at 07 42 35

Screenshot 2022-10-25 at 07 43 13
</details>

### Related issues, RFCs

**Issue number:** #1132 

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [ ] I have *commented* my code where necessary, particularly in areas that should be flagged with a TODO, or hard-to-understand areas
- [x] I have made corresponding changes to the *documentation*
- [ ] I have made corresponding changes to the *examples*
- [x] My changes generate *no new warnings*
- [x] The *code coverage* hasn't decreased
- [ ] I have *added tests* that prove my change is effective and works
- [x] New and existing *unit tests pass* locally and in Github Actions
- [ ] Any *dependent changes have been merged and published* in downstream module
- [x] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by heitorlessa** [comment]
reading...

File: docs/index.md [hunk 1]
```diff
@@ -67,206 +57,172 @@ You can include Lambda Powertools Lambda Layer using [AWS Lambda Console](https:
...
-
-
-Or refer to the installation guide of each utility: 
-
-[Installation guide for the **Tracer** utility](./core/tracer.md#getting-started)
-
-[Installation guide for the **Logger** utility](./core/logger.md#getting-started)
-
-[Installation guide for the **Metrics** utility](./core/metrics.md#getting-started)
+	Lambda Powertools Lambda Layer do not include `pydantic` library - required dependency for the `parser` utility. See [SAR](#sar) option instead.
```
Comment (heitorlessa) [inline docs/index.md:225]:
```suggestion
```

removing as this is a leftover from Python docs

**Review by heitorlessa — CHANGES_REQUESTED** [review]
LGTM, only one part to remove (leftover from Python docs)

**Comment by dreamorosi** [comment]
@heitorlessa thank you for the review & for spotting the leftover line about pydantic. 

I have committed your suggestion!

**Review by heitorlessa — APPROVED** [review]

**Pull request merged**